src/tests/requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-apache-airflow==2.5.3
+apache-airflow==2.8.0
```

Comment:
@kinjalGakhani Composer does not support 2.8.0 at present, is there a reason we need to update it?

Reply:
@dmedora - Unit test was failing and because of it I made this change (which is also the solution of unit test PR # 24)

Failed unit test reference - https://pantheon.corp.google.com/cloud-build/builds;region=global/6a100a49-e016-4e34-a4a1-4949dc0e0bff;step=3?e=PangolinKitchenLaunch::PangolinKitchenEnabled&mods=bigquery_admin_offline_translation&project=pso-data-migration-tool-test

Reply:
Ok, I will comment on PR#24 to ask there as well. To me it doesn't seem like an ideal solution because then we're not testing if it will run in the actual environment anymore (ie. if this fails <2.8.0, then in Composer it will fail).

Reply:
Hi @dmedora , this one is sorted in the both PR, please re verify and let me know in case of any queries, thanks !!